maidsafe/safe_browser
# fix(Invite): Invite page not notified as blocked when opening in clea…

QA:

Click on the invite link. The page should open in the clearnet browser, and there should be no warning about the blocked url.

Sadly the tab still hangs around, but that's the standard behaviour we have here atm.

This is updated for dev and w/ @hunterlester 's comments seen to (from https://github.com/maidsafe/safe_browser/pull/694)
